content/full-day/6-code.md

```diff
@@ -78,18 +78,22 @@ Adding the filters to the page will require updating a minimum of three files -
    - **server/app.py**
    - **server/test_app.py**
    - **client/src/components/DogList.svelte** 
-2. Select **Copilot Edits** in the Copilot Chat window.
-3. Ensure **Claude 3.7 Sonnet** is selected for the model.
-4. Select **Add Context...** in the chat window and **Open Editors** from the prompt. This will add all currently open files to the context.
-5. Ask Copilot to perform the operation you want, to update the page to add the filters. It should meet the following requirements:
+2. Open GitHub Copilot Chat.
+3. Switch to edit mode by selecting **Edit** in the chat mode dropdown at the bottom of Chat view (should be currently **Ask**)
+4. Ensure **Claude 3.7 Sonnet** is selected for the model.
+5. Select **Add Context...** in the chat window.
+6. Select **server/app.py**, **client/src/components/DogList.svelte** and **server/test_app.py** files (you need to select **Add context** for each file) 
+  - **Hint**: if you type the file names, they will show up in the filter. You can also drag the files or right click file in explorer and select `Copilot -> Add File to Chat`)
```

Comment:
Rather than separate bullet point let's have this as a note if we're going to include drag & drop. I'd also highlight this doesn't need to be all-inclusive of every option available

Reply:
Do you prefer the note to be "inside" the bullet point 6), or after all bullet points?

Reply:
Let's go with "inside" bullet point 6

Reply:
It will also make @peckjon happy as he wanted a callout to this previously. I personally never use the feature, but since there's now a second request apparently I'm the oddball

Reply:
Updated in 474fddb7d9810c33beeb75db99e448a214d44d0e